README.md

```diff
@@ -1,2 +1,45 @@
-# Tic_Tac_Toe
-use ruby best practices to create a 3x3 Tic Tac Toe game
+# 3x3Tic_Tac_Toe
+
+# Game Description
+
+- its a two player board game. The board consists of 9 cells (3x3). player 1 goes with 'x' for example then player 2 goes with 'o'. if any of the two player can sit a 3 consecutive x's or o's either horizontal ,vertical or even diagonal ,then this player considered as the winner.
+
+# Technologies Used
+
+- Ruby
+
+# RULES FOR TIC-TAC-TOE.
+- The game is played on a grid that's 3 squares by 3 squares.
+- You are X, your friend is O. You are O, your friend is X.
+- The first player to get 3 of his/her marks in a row (up, down, across, or diagonally) is the
+winner.
+- When all 9 squares are full, the game is over.
+- if no player get 3 of his/her marks in a row, The game is draw.
+
+# Game Preview
+
+
+![Game preview](gamepreview.gif)
+
+
+# Install Instruction
+
+1. Clone the Repo on you computer
+2. make sure you have Ruby installed; use 'Ruby -v' to check for Ruby preexistence.
```

Comment:
Instead of just saying clone the repo. It's better when you mention how to clone.

like `git clone https://github.com/Abdona/Tic_Tac_Toe.git`

and on line no. `29` Instead of just writing **go to the rebo location folder** you can also show how to go like this `cd your-repo-name`. Bdw, there is a typo with repo on line 29 as well.

kindly fix this before merging.